au4/src/au3wrap/au3wrapmodule.cpp

```diff
@@ -52,6 +57,10 @@ void Au3WrapModule::onInit(const muse::IApplication::RunMode&)
 
     m_audioDevicesProvider->init();
 
+    //! TODO AU4: we probably want to have preferences for temporary directory later
```

Comment:
by default, it is `GlobalConfiguration::userAppDataPath` or last selected folder when saving previous project